doc/api/path.md

```diff
@@ -2,31 +2,86 @@
 
     Stability: 2 - Stable
 
-This module contains utilities for handling and transforming file
-paths. The file system is not consulted to check whether paths are valid.
+The `path` module provides utilities for working with file paths. It can be
+accessed using:
 
-Use `require('path')` to use this module.  The following methods are provided:
+```js
+const path = require('path');
+```
+
+The file system is not consulted to determine if paths are valid.
```

Comment:
While this statement is generally true, `path.resolve()` *does* make use of `process.cwd()` in some cases and `process.cwd()` *can* fail if the cwd no longer exists for example.